nrf_security/src/drivers/nrf_cc3xx/CMakeLists.txt

```diff
@@ -3,32 +3,34 @@
 #
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
+get_target_property(NRF_CC3XX_MBEDCRYPTO_INCLUDE_PATH mbedcrypto_cc3xx_imported INTERFACE_INCLUDE_DIRECTORIES)
 
-add_library(cc3xx_mbedcrypto STATIC IMPORTED)
-add_library(cc3xx_psa_crypto STATIC IMPORTED)
-
-if(CONFIG_CC3XX_RELEASE_VERSION)
-set_property(TARGET cc3xx_mbedcrypto PROPERTY
-              IMPORTED_LOCATION "${NRF_SECURITY_ROOT}/../../sdk-nrf-cc3xx/build/release-0.9.11-out/external/nrf_cc312_mbedcrypto/lib/cortex-m33/soft-float/no-interrupts/libnrf_cc312_mbedcrypto_0.9.11.a")
+if(CONFIG_SOC MATCHES "nRF53")
 set_property(TARGET cc3xx_psa_crypto PROPERTY
-             IMPORTED_LOCATION "${NRF_SECURITY_ROOT}/../../sdk-nrf-cc3xx/build/release-0.9.11-nrf5340/libcc3xx_psa_crypto.a")
+             IMPORTED_LOCATION "${CMAKE_CURRENT_LIST_DIR}/lib/nrf5340/libcc3xx_psa_crypto.a")
+elseif(CONFIG_SOC MATCHES "nRF91")
+  set_property(TARGET cc3xx_psa_crypto PROPERTY
+               IMPORTED_LOCATION "${CMAKE_CURRENT_LIST_DIR}/lib/nrf9160/libcc3xx_psa_crypto.a")
 else()
-set_property(TARGET cc3xx_mbedcrypto PROPERTY
-              IMPORTED_LOCATION "${NRF_SECURITY_ROOT}/../../sdk-nrf-cc3xx/build/debug-0.9.11-out/external/nrf_cc312_mbedcrypto/lib/cortex-m33/soft-float/no-interrupts/libnrf_cc312_mbedcrypto_0.9.11.a")
 set_property(TARGET cc3xx_psa_crypto PROPERTY
-             IMPORTED_LOCATION "${NRF_SECURITY_ROOT}/../../sdk-nrf-cc3xx/build/debug-0.9.11-nrf5340/libcc3xx_psa_crypto.a")
+             IMPORTED_LOCATION "${CMAKE_CURRENT_LIST_DIR}/lib/nrf52840/libcc3xx_psa_crypto.a")
 endif()
 
 # Link to library containing PSA Crypto Driver
 target_link_libraries(${mbedcrypto_target}
-  PRIVATE
+  INTERFACE
```

Comment:
this changes the linking of `${mbedcrypto_target}` to actually link with below libraries, and not passing this knowledge to any library linking to `${mbedcrypto_target}`, to instead, any library linking with `${mbedcrypto_target}` will now link to below libraries, but `${mbedcrypto_target}` will no longer do so.

This looks unsafe, unless a bug has been discovered.
Please check that this is correct, and update commit message as to why this is safe.
I can't decipher the reason based on current commit message.